smart-contracts/contracts-common/concordium-contracts-common/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Unreleased changes
 
 - `HashBytes::new` is now a `const` function.
+- Derive `serde::Serialize` and `serde::Deserialize` for `AccountSignature`, `CredentialSignature`, `Signature`, and `SignatureEd25519` when feature `derive-serde` is enabled.
+- Implement `serde::Serialize` and `serde::Deserialize` for a fixed byte array of length 64 in the helper function `fixed_byte_array_64_hex` when feature `derive-serde` is enabled.
```

Comment:
This function is not exposed as I see it.

Reply:
removed it from the changelog